site/_includes/partials/sample-item.njk

```diff
@@ -0,0 +1,66 @@
+{% from 'macros/icon.njk' import icon with context %}
```

Comment:
Please rename to `extension-sample-item.njk`, so it's easier to spot where it is used.